service/src/test/java/uk/gov/hmcts/reform/fpl/service/NoticeOfProceedingsServiceTest.java

```diff
@@ -104,8 +105,8 @@ void shouldApplySentenceFormattingWhenMultipleChildrenExistOnCase() {
                 .build())
             .build();
 
-        Map<String, Object> templateData = noticeOfProceedingService.getNoticeOfProceedingTemplateData(caseData);
```

Comment:
looks like we still have `Map` in the import list. checkstyle will complain here

Reply:
I didn't finish working on this yet. I have parked this to work on a bug. You are right, i will fix it.